### PR TITLE
Improve readme.

### PR DESCRIPTION
**Pull request by Thom1729**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
 # sublime_lib
 
-Utility library for frequently used functionality in Sublime Text library and convenience functions or classes
+An unofficial "standard library" for Sublime Text. Provides a variety of useful convenience features for other packages.
```
Comment (FichteFoll) [inline README.md]: I don't think we need to declare it as "unofficial" when there is no official competition. Not sure if "standard library" is fitting either.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+
+Then, anywhere in your package, you can import sublime_lib by name:
+
+```python
+import sublime_lib
+```
+
+## Features
+
+For complete documentation of all features, see the [API documentation](https://sublimetext.github.io/sublime_lib/).
```
Comment (FichteFoll) [inline README.md]: I highly suggest adding an opinionated short list of key features that the library provides. Having to go to yet another page only to find out what this thing actually *does* is not good potential user engagement.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,27 @@
 # sublime_lib
 
-Utility library for frequently used functionality in Sublime Text library and convenience functions or classes
+An unofficial "standard library" for Sublime Text. Provides a variety of useful convenience features for other packages.
+
+## Installation
+
+To make use of sublime_lib in your own package, first declare it as a [dependency](https://packagecontrol.io/docs/dependencies) of your package. Create a file named `dependencies.json` in the root of your package with the following contents:
```
Comment (FichteFoll) [inline README.md]: Might need to mention running the Satisfy Dependencies command.

File: README.md (hunk 2 above)
Comment (Thom1729) [reply README.md]: Agreed.

**Pull request merged**